### PR TITLE
Fix swagger export empty

### PR DESCRIPTION
**Pull request by cuminandpaprika**
Fixes https://github.com/anz-bank/sysl/issues/794, where empty types using ... notation in Sysl are unhandled by the Sysl Exporter.

Changes proposed in this pull request:
- Add support for empty types in sysl export
- Remove schema references in endpoint definitions 
    -  For return statement:`Return Empty`
    - For return statement:`Return 200 <: Empty`
    - For return statement: `Return 200`, where 200 can be any status code
- Document current workaround implementation

Checklist:
- [x] Added related tests
- [x] Made corresponding changes to the documentation

### CODE REVIEW

File: pkg/exporter/test-data/SWAGGER_WITH_EMPTY.sysl [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+testapp "foo" [package="package_foo"]:
+    @version = "0.0.0"
+    @description =:
+        | foo
+
+    /test:
+        GET:
+            | No description.
+            return 200 <: Empty
```
Comment (nofun97) [inline pkg/exporter/test-data/SWAGGER_WITH_EMPTY.sysl:15]: this doesn't look right, isn't it usually `... <: 200`?

Comment (ashwinsajiv) [reply pkg/exporter/test-data/SWAGGER_WITH_EMPTY.sysl:15]: Isn't <: this operator defining the left to be a subset of the right? 
So in that case aren't the two totally different?

Comment (cuminandpaprika) [reply pkg/exporter/test-data/SWAGGER_WITH_EMPTY.sysl:15]: Yeah I'm kinda confused, what does `... <: `200 mean?

Comment (cuminandpaprika) [reply pkg/exporter/test-data/SWAGGER_WITH_EMPTY.sysl:15]: Also, just added handling of some extra cases which mean empty:
- Response code without a return Type
- Return type is a response code plus ...
- Return type is ...
- Return type is just Empty
```

    /testResponseCodeEmpty:
        GET:
            | No description.
            return 200 <: Empty

    /testResponseCodeResponse:
        GET:
            | No description.
            return 200

    /testResponseCodeDots:
        GET:
            | No description.
            return 200 <: ...

    /testDots:
        GET:
            | No description.
            return ...

    /testDots:
        GET:
            | No description.
            return Empty

    !type Empty:
        ...

```

Comment (nofun97) [reply pkg/exporter/test-data/SWAGGER_WITH_EMPTY.sysl:15]: @cuminandpaprika ah sorry I mean `variable <: 200`, I don't think I've seen `200 <: type` (with type being `Empty` in this case).

Comment (cuminandpaprika) [reply pkg/exporter/test-data/SWAGGER_WITH_EMPTY.sysl:15]: No worries, fixed up remove support of ... in endpoints!

**Pull request merged**